chatmm/src/main/java/com/likeminds/chatmm/dm/viewmodel/DMFeedViewModel.kt

```diff
@@ -55,12 +58,20 @@ class DMFeedViewModel @Inject constructor(
         data class CheckDMStatus(val errorMessage: String?) : ErrorMessageEvent()
     }
 
+    private val chatRoomListShimmerView by lazy {
+        HomeChatroomListShimmerViewData.Builder().build()
+    }
+
     fun isDBEmpty(): Boolean {
         return (lmChatClient.getDBEmpty().data?.isDBEmpty ?: false)
     }
 
     fun syncChatrooms(context: Context): Pair<LiveData<MutableList<WorkInfo>>?, LiveData<MutableList<WorkInfo>>?>? {
-        return lmChatClient.syncChatrooms(context)
+        return lmChatClient.loadDMChatrooms(context)
+    }
+
+    fun setWasChatroomFetched(value: Boolean) {
+        homeFeedPreferences.setShowDMFeedShimmer(value)
```

Comment:
change the name of `setShowDMFeedShimmer` as it is working opposite to what the name suggests